src/components/form-slice/FormPartRoute/index.tsx

```diff
@@ -0,0 +1,161 @@
+import { useFormContext } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { FormItemNumberInput } from '@/components/form/NumberInput';
+import { FormItemSwitch } from '@/components/form/Switch';
+import { FormItemTagsInput } from '@/components/form/TagInput';
+import { FormItemTextarea } from '@/components/form/Textarea';
+import { FormItemTextInput } from '@/components/form/TextInput';
+import { NamePrefixProvider } from '@/utils/useNamePrefix';
+import { FormItemPlugins } from '../FormItemPlugins';
+import { FormPartBasic } from '../FormPartBasic';
+import { FormPartUpstream, FormSectionTimeout } from '../FormPartUpstream';
+import { FormSection } from '../FormSection';
+import { Divider, InputWrapper } from '@mantine/core';
+import type { RoutePostType } from './schema';
+import { FormItemSelect } from '@/components/form/Select';
+import { APISIX } from '@/types/schema/apisix';
+
+const FormPartBasicWithPriority = () => {
+  const { t } = useTranslation();
+  const { control } = useFormContext<RoutePostType>();
+  return (
+    <FormPartBasic>
+      <FormItemNumberInput
+        control={control}
+        name="priority"
+        label={t('form.route.priority')}
+      />
+      <FormItemSelect
+        control={control}
+        name="status"
+        label={t('form.route.status')}
+        data={APISIX.RouteStatus.options.map((v) => v.value.toString())}
```

Comment:
I'm okay with this option list, but any plans to add an `info` icon and description for this field?

<img width="1107" alt="image" src="https://github.com/user-attachments/assets/e09f0326-2771-43ef-9426-c9052accb0fa" />

Reply:
After all pages are completed...

Reply:
After all, it's strictly aligned with the API now.